server/channels/api4/data_retention.go

```diff
@@ -264,9 +264,10 @@ func searchTeamsInPolicy(c *Context, w http.ResponseWriter, r *http.Request) {
 func addTeamsToPolicy(c *Context, w http.ResponseWriter, r *http.Request) {
 	c.RequirePolicyId()
 	policyId := c.Params.PolicyId
-	teamIDs, err := model.SortedArrayFromJSON(r.Body, *c.App.Config().ServiceSettings.MaximumPayloadSizeBytes)
-	if err != nil {
-		c.Err = model.NewAppError("addTeamsToPolicy", model.PayloadParseError, nil, "", http.StatusBadRequest).Wrap(err)
+	var teamIDs []string
+	jsonErr := json.NewDecoder(r.Body).Decode(&teamIDs)
```

Comment:
Similar question. Why are we using the json decoder instead of the ArrayFromJSON? Is it due to the need for having them sorted? 

If there's a reason for the json decoder usages, should we call `model.RemoveDuplicateStrings` to remove any duplicates?